features/com.itemis.xtext.testing.feature/feature.xml

```diff
@@ -2,7 +2,7 @@
 <feature
       id="com.itemis.xtext.testing.feature"
       label="Xtext Testing Framework"
-      version="0.2.0.qualifier">
+      version="0.3.0.qualifier">
```

Comment:
is there a specific reason to bump version?

Reply:
My idea is here: Somewhen a version 0.2.0 has been around, and this is now a change to what was available then. Semantic versioning FTW.

Reply:
Like you prefer... I could roll it back if you want.

Reply:
we use the 0.2.0 all the time. so never did semantic version.
i also assume that this enhances comp. not lowers it